docs/data/capture/performance-metrics.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Performance Monitoring Data Capture"
+linkTitle: "Monitor Performance"
+description: "Capture and sync data about your machines' performance."
+weight: 12
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+icon: true
+images: ["/data/monitor.gif"]
+videos: ["/data/monitor.webm", "/data/monitor.mp4"]
+aliases:
+  - "/services/data/capture/"
+# SME: Alexa Greenberg
+---
+
+You can capture and monitor the following metrics about the performance of individual machines or your entire fleet:
+
+- **Wireless Signal Strength and Quality**: signal level, link quality, and noise level
+- **Memory Usage**: memory statistics, including total available memory, used percentage, and specifics on various types of memory (cached, free, slab, etc.)
+- **CPU Utilization**: CPU usage across different states (user, system, idle, etc.)
+- **Disk I/O**: metrics on read and write operations, including bytes transferred and operation times
+- **Network Traffic**: detailed network statistics, including bytes sent and received, packet information, and error counts, provide a deep dive into a device's network performance
+
+## Requirements
+
+To capture data from a machine, you need to [create a machine in Viam](https://docs.viam.com/manage/fleet/machines/#add-a-new-machine) and follow the setup instruction to install `viam-server` and connect it to the Viam app.
+
+{{< alert title="Note" color="note" >}}
+You must run `viam-server` with `sudo` to monitor machine performance metrics.
+{{< /alert >}}
+
+## Add sensor to measure performance metrics
+
+To obtain performance metrics about your machine, use the [`viam-telegraf-sensor`](https://app.viam.com/module/viam/viam-telegraf-sensor) module from the [Viam registry](/registry/).
+The module provides a `sensor` that allows you to obtain readings containing your machine's performance metrics.
+
+1. Go to your machine's **Config** page and click **Create component**.
+2. Then select the `viam:viam-sensor:telegrafsensor` model from the [`viam-telegraf-sensor` module](https://app.viam.com/module/viam/viam-telegraf-sensor).
+3. Click **Add module**, then enter a name for your sensor, for example `my-telegrafsensor`, and click **Create**.
+
+In the next step you will configure the data manager to capture and sync the performance metrics from your configured sensor.
```

Comment:
(opt) I would veer on the side of referring to it as data management service consistently. Also not sure if this sentence is necessary although it does provide a bit of a transition. Maybe consider line 43 before 41?